clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -360,8 +360,10 @@ func (w *waitForControlledPodsRunningMeasurement) updateOpResourceVersion(runtim
 func (w *waitForControlledPodsRunningMeasurement) getObjectCountAndMaxVersion() (int, uint64, error) {
 	var desiredCount int
 	var maxResourceVersion uint64
-	objects, err := runtimeobjects.ListRuntimeObjectsForKind(w.clusterFramework.GetClientSets().GetClient(),
-		w.kind, w.selector.Namespace, w.selector.LabelSelector, w.selector.FieldSelector)
+	objects, err := runtimeobjects.ListRuntimeObjectsForKind(
+		w.clusterFramework.GetClientSets().GetClient(),
+		w.clusterFramework.GetDynamicClients().GetClient(),
```

Comment:
just pass MultiClientSet instead of passing both

Reply:
As discussed offline, I would need to pass both MultiClientSet and MutliDynamicClient, so it's better to keep it this way.